common/src/main/java/com/redhat/cloud/notifications/models/WebhookProperties.java

```diff
@@ -32,10 +35,25 @@ public class WebhookProperties extends EndpointProperties {
     @JsonProperty("secret_token")
     private String secretToken; // TODO Should be optional
 
+    /**
+     * The ID of the "secret token" secret in the Sources backend.
+     */
+    @Column(name = "secret_token_id")
+    @JsonIgnore
+    private long secretTokenSourcesId;
+
     @Convert(converter = BasicAuthenticationConverter.class)
     @JsonProperty("basic_authentication")
+    @Valid
     private BasicAuthentication basicAuthentication;
 
+    /**
+     * The ID of the "basic authentication" secret in the Sources backend.
+     */
+    @Column(name = "basic_authentication_id")
+    @JsonIgnore
+    private long basicAuthenticationSourcesId;
```

Comment:
Same as above - we will need to do the same for the other fields that we are adding

```suggestion
    private Long basicAuthenticationSourcesId;
```

Reply:
Done!